pages/index.js

```diff
@@ -48,6 +49,19 @@ const HomePage = () => (
         display: block;
       }
     `}</style>
+    <Head
+      title={"Sketchbook.js: the web design tool that lives in your repo."}
```

Comment:
As per #1, let make this just:

```suggestion
      title={"Sketchbook.js"}
```